frontend/src/common/models/user/IUser.ts

```diff
@@ -6,4 +8,5 @@ export interface IUser {
   imageUrl?: string;
   title?: string;
   status?: string;
+  workspaces: IWorkspaceResponse[];
```

Comment:
you have to create separate interface like in backend

Reply:
Done

Reply:
waiting for changes